packages/strapi-plugin-upload/admin/src/components/FiltersList/utils/formatFilters.js

```diff
@@ -0,0 +1,31 @@
+const formatFilters = filters => {
```

Comment:
This utility is not necessary if you handle the `file` filter directly and you just update the sent params according to `mime_contains=file`